stable/localvolumeprovisioner/values.yaml

```diff
@@ -1 +1,4 @@
-storageclassdefault: true
+storageclass:
```

Comment:
This change isn't compatible with what is checked for in https://github.com/mesosphere/kubeaddons/blob/master/pkg/utils/supported_addons.go#L348